app/controllers/users_controller.rb

```diff
@@ -5,6 +5,23 @@ def index
 
   def show
     @user = User.find(params[:id])
+    @movies = {}
+    @movie_images = {}
+    @hosted_parties = []
+    @invited_parties = []
+
+    @user.viewing_parties.each do |party|
+      if party.party_guests.find_by(user_id: @user.id, host: true)
+        @hosted_parties << party
+      else
+        @invited_parties << party
+      end
+
+      movie_id = party.movie_id
+      movie = MoviesService.new.find_movie(movie_id)
+      @movies[movie_id] = movie.title
+      @movie_images[movie_id] = movie.image
+    end
```

Comment:
Sending only @user alone into the view is all we need-
The user has the viewing parties associated with it, and then each viewing party is connected to the movie and has the host vs invitee info, and each party likewise has the movie photo and title info as well.

Then within the view something like: 
```
<% @user.viewing_parties.each do |party| %>
        <%= party.movie_title (with link to show page) %>
        <%= party.movie_image %>
        <%= party.date %>
        <%= party.time %>
         <%= "Hosting" if @user == party.host %>
        <%= "Invited" if @user != party.host %>
       <%= party.host if @user != party.host %>
         <%= party.party_guests %> (with the code that makes the Host's name bold)
<% end %>
```

Reply:
Done